### DISCUSSION TITLE
Sleep Mode

### DISCUSSION THREAD

**Discussion by QuadrifoglioVerde** [jmcollin78/versatile_thermostat#253] [Ideas]
Can you please add another mode for "sleep" mode?

**Comment by jmcollin78** [comment]
Hello, as explained many and many times I will not do that. Each users have its own way if using and it is not possible to take all needs into account. Some wants confort-1, confort-2, some wants Sleep, ... 

I will add a anti-freezing (don't exactly the term in English - we said hors gel in French) but that is all planed yet.

The goal is to keep it simple. You always have the possiblity to control you VTherm in manual mode if presets are not enough.

But, consider the fact that Sleep mode is Eco and that's it.
